src/ScriptCs.Core/ScriptLibraryWrapper.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using ScriptCs.Contracts;
+
+namespace ScriptCs
+{
+    public abstract class ScriptLibraryWrapper
```

Comment:
In my opinion it should be there, though it will be a breaking change if we add it to the interface. It will only implement those who have a custom host though.

Today custom scriptcs like "ScriptCs.Logger" do rely on this to get the args, so maybe we should add it.